cmd/cost/list.go

```diff
@@ -281,7 +368,11 @@ func (o OUCost) printCostPerAccount(awsClient awsprovider.Client) {
 	}
 	if o.options.csv {
 		if o.options.sum {
-			fmt.Printf("%s,%s,%s,%s\n", *o.OU.Id, "SUM", sum.StringFixed(2), unit)
+			output := fmt.Sprintf("%s,%s,%s,%s", *o.OU.Id, "SUM", sum.StringFixed(2), unit)
+			if o.options.claims {
+				output = ",," + output
```

Comment:
Are `,,` as output intentional?

Reply:
Would it make sense to add the format as a comment?

Reply:
it's CSV generation. Not very beautiful, but works.